src/layout.sh

```diff
@@ -158,27 +157,27 @@ start_listener() {
   __recalculate_layout() { run_layout $layout $args 2> /dev/null || true; }
 
   # Then listen to node changes and recalculate as required
-  bspc subscribe node_{add,remove,transfer,flag,state} desktop_focus | while read line; do
-    event=$(echo "$line" | awk '{print $1}')
-    arg_index=$([[ "$event" == "node_transfer" ]] && echo "6" || echo "3")
-    desktop_id=$(echo "$line" | awk "{print \$$arg_index}")
+  bspc subscribe node_{add,remove,transfer,flag,state} desktop_focus | while read -a line; do
```

Comment:
Didn't know you could read it as an array of words. This is pretty neat!

Reply:
with bash `read` you can do much more